tests/test_settings.py

```diff
@@ -1,24 +1,51 @@
 # -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
 import pytest
 
 
 @pytest.mark.usefixtures('coveragerc')
-def test_project_currencies(testdir):
-    testdir.makepyfile(test_settings='''
-    INSTALLED_APPS = ['djmoney']
-    CURRENCIES = ['USD', 'EUR']
-    SECRET_KEY = 'foobar'
-    ''')
-    testdir.makepyfile('''
-    def test_project_currencies():
-        from djmoney.settings import CURRENCY_CHOICES
-
-        assert CURRENCY_CHOICES == [('EUR', 'Euro'), ('USD', 'US Dollar')]
-    ''')
-    result = testdir.runpytest_subprocess(
-        '--verbose', '-s',
-        '--ds', 'test_settings',
-        '--cov', 'djmoney',
-        '--cov-config', 'coveragerc.ini',
-    )
-    assert 'test_project_currencies.py::test_project_currencies PASSED' in result.stdout.lines
+class TestCurrencies:
+
+    def assert_test_passed(self, testdir, lines):
+        result = testdir.runpytest_subprocess(
+            '--verbose', '-s',
+            '--ds', 'app_settings',
+            '--cov', 'djmoney',
+            '--cov-config', 'coveragerc.ini',
+        )
+        result.stdout.fnmatch_lines(lines)
```

Comment:
Ah, I see what you mean after reading this in the docs:

> Finally, avoid aliasing your settings as module-level constants as override_settings() won’t work on such values since they are only evaluated the first time the module is imported.

Maybe we can use `override_settings` + `reload(djmoney.settings)` ?